client/signup/config/flows.js

```diff
@@ -171,11 +171,16 @@ function getCurrentFlowNameFromTest( currentURL ) {
 		return ( 'noSurvey' === abtest( 'verticalSurvey' ) ) ? 'main' : 'verticals';
 	}
 
-	// Only consider users from the general /start path.
+	// Only consider users from the WordPress.com homepage for the DSS test.
 	if ( '/start/en?ref=homepage' === currentURL && 'dss' === abtest( 'dss' ) ) {
 		return 'dss';
 	}
 
+	// Consider remaining homepage users for the Triforce AB test.
+	if ( '/start/en?ref=homepage' === currentURL && 'notTested' === getABTestVariation( 'dss' ) && 'triforce' === abtest( 'triforce' ) ) {
```

Comment:
Nice! One more comment here: just to prevent us from making a mistake later, I think the `notTested` check for DSS should also accept `null` or `undefined`, so that if we remove the DSS test we don't accidentally disable this one. (That would happen because a user who didn't get `abtest('dss')` called would never end up in the `notTested` group.)